pkg/hubble/parser/threefour/parser.go

```diff
@@ -620,3 +622,30 @@ func decodeDebugCapturePoint(dbg *monitor.DebugCapture) pb.DebugCapturePoint {
 	}
 	return pb.DebugCapturePoint(dbg.SubType)
 }
+
+func decodeNetworkInterface(tn *monitor.TraceNotify, dbg *monitor.DebugCapture) *pb.NetworkInterface {
+	ifIndex := 0
+	if tn != nil {
+		ifIndex = int(tn.Ifindex)
+	} else if dbg != nil {
+		switch dbg.SubType {
+		case monitor.DbgCaptureDelivery,
+			monitor.DbgCaptureFromLb,
+			monitor.DbgCaptureAfterV46,
+			monitor.DbgCaptureAfterV64,
+			monitor.DbgCaptureSnatPre,
+			monitor.DbgCaptureSnatPost:
+			ifIndex = int(dbg.Arg1)
+		}
+	}
+
+	if ifIndex == 0 {
+		return nil
+	}
+
+	name, _ := link.GetIfNameCached(ifIndex)
```

Comment:
err ok to ignore here?

Reply:
Yes. I added a comment to indicate that it is intentional.